src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -3,6 +3,8 @@ language_settings:
   java:
     package_name: com.google.gcloud.pubsub.spi
     release_level: BETA
+    class_names:
+      google.example.library.v1.LibraryService: LibraryClient
```

Comment:
What is the major use case of this field? Is it only for renaming interface names or any class names?